CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
BLOCKER COMMENT for @Scienfitz Review since the files have changed quite dramatically and I would like another review just to be sure that everyone is happy.

Reply:
resolving as vote was reset